apps/infrastructure/src/acm.tf

```diff
@@ -11,4 +12,34 @@ module "acm" {
   wait_for_validation = true
   validation_method   = "DNS"
   dns_ttl             = 3600
+}
+
+## Certificate HTTPS for Cloudfront Websites
+provider "aws" {
+  alias  = "us-east-1"
+  region = "us-east-1"
+}
```

Comment:
What do you think to move this provider into the [`main.tf` file](https://github.com/pagopa/b2b-portals/blob/855f0ad944469d96fdee18dac09e8893640f6352/apps/infrastructure/src/main.tf#L14-L19)?

[There already is a provider there](https://github.com/pagopa/b2b-portals/blob/855f0ad944469d96fdee18dac09e8893640f6352/apps/infrastructure/src/main.tf#L14-L19), so I think it would be better to keep the providers in the same file.

Reply:
provider moved to main